components/Filter.js

```diff
@@ -1,7 +1,14 @@
 import React from "react";
 
-const Filter = () => {
-	return <div>Filter</div>;
+const Filter = ({ topic }) => {
+	return (
+		<div className='w-[270px] h-[55px] flex justify-between bg-black rounded-full'>
```

Comment:
Avoid hard coded pixel values, it causes undefined and unpredictable behavior with different screen sizes. instead use percentages